qa/automation/CareerPortal.Tests.Scenarios/1. Search by Filters/1.1. Search by Filters.feature

```diff
@@ -0,0 +1,134 @@
+Feature: 1.1. Search by Filters
+
+Background: 
+    Given User is on https://career.quantori.com website
+    And There are 10 available <positions> on the jobs page
+    And There are 2 available <competence> on the jobs page
+    And There are 2 available <technologies> on the jobs page
+    |Position                                              ||competence       ||technologies|
+    |Junior JavaScript Engineer                            ||Databases        ||SQL         |
+    |Senior/Lead Manual QA Engineer (Life Science projects)||Business Analysis||.NET        |
+    |Senior/Lead Manual QA Engineer (internal projects)    ||                 ||            |
+    |Data Engineer (SQL)                                   ||                 ||            |
+    |Snowflake Data Warehouse Certified Engineer           ||                 ||            |
+    |Senior DWH / ETL Data Engineer                        ||                 ||            |
+    |Data Engineer (Spark)                                 ||                 ||            |
+    |Junior JavaScript Engineer                            ||                 ||            |
+    |Senior QA Automation (Python) Engineer / SDET         ||                 ||            |
+    |Senior Data Engineer                                  ||                 ||            |
+
+
+Scenario: 1.1_01 Checking the function of the search button
+    When User pushes "search" button
+    Then User should be redirected to the new page "https://career.quantori.com/ru/positions"
+    And  User should see default page "Quantori open job positions"
+
+Scenario: 1.1_02 Returning to the general job list from the "Quantori Hot Open Job Positions list"
+    Given User opened https://career.quantori.com/ru/positions/cloud-python-engineer website
+    When At the bottom of the page user push "Back to open job positions" button
```

Comment:
First goes action and then description where it is, so better - When user clicks on "Back to open job positions" button at the bottom of the page

Reply:
You can also return back by clicking on "Back" button so we can make this test with Examples section-
When user clicks on "<name>" button

Examples:
| button |
|  Back to open job positions |
| Back |